database/neo4j/README.md

```diff
@@ -0,0 +1,11 @@
+# neo4j
```

Comment:
Add docs about installing `seabolt` and statically linking `OpenSSL` to the `README`